docs/Vue.js/DeclarativeRendering.md

```diff
@@ -0,0 +1,64 @@
+---
+id: Vue.js Declarative Rendering
```

Comment:
reploace `id: Vue.js Declarative Rendering` to `id: vue.js-declarative-rendering`